php/PHPCD.php

```diff
@@ -2,7 +2,19 @@
 
 class PHPCD extends RpcServer
 {
-    public function info($class_name, $pattern, $mode) {
+    /**
+     *  @param array Map between modifier numbers and displayed symbols
+     */
+    private $modifierSymbols = [
```

Comment:
On the one hand, I kept the present order ("$modifier $static"), on the other - I follow PSR-2 suggestion concerning order (https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#1-overview).